q2_types/feature_data/_transformer.py

```diff
@@ -364,6 +364,27 @@ def _33(ff: AlignedDNAFASTAFormat) -> qiime2.Metadata:
     return _dnafastaformats_to_metadata(ff)
 
 
+@plugin.register_transformer
+def _34(ff: AlignedDNAFASTAFormat) -> pd.Series:
+    return _dnafastaformats_to_series(ff)
+
+
+@plugin.register_transformer
+def _35(data: pd.Series) -> AlignedDNAFASTAFormat:
+    ff = AlignedDNAFASTAFormat()
+    with ff.open() as f:
+        for id_, seq in data.iteritems():
+            sequence = skbio.DNA(seq, metadata={'id': id_})
+            skbio.io.write(sequence, format='fasta', into=f)
```

Comment:
This block shows up more than once in this file - might be a good candidate for a util function.